geomesa-filter/src/main/scala/org/locationtech/geomesa/filter/PropertyLiteral.scala

```diff
@@ -6,7 +6,7 @@
  * http://www.opensource.org/licenses/apache2.0.php.
  */
 
-package org.locationtech.geomesa.accumulo.filter
+package org.locationtech.geomesa.filter
```

Comment:
Should this move between sub-modules?  (I haven't checked the full path quite yet.)